validation-test/Sema/type_checker_perf/slow/rdar30596744_2.swift.gyb

```diff
@@ -1,4 +1,4 @@
-// RUN: %scale-test --invert-result --begin 1 --end 5 --step 1 --select incrementScopeCounter %s --expected-exit-code 1
+// RUN: %scale-test --invert-result --begin 1 --end 4 --step 1 --select incrementScopeCounter %s --expected-exit-code 1
```

Comment:
Why did you change this?

Reply:
Cause scalw test can’t fit the curve with these changes unfortunately...